talktorials/1_ChEMBL/T1_ChEMBL.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>There are some fields named <code>standard_*</code> that already provide this unit normalization. Is this being discussed in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I did not know that. In that case, I would drop the whole conversion part and use <code>standard_value</code>  and <code>standard_unit</code>  instead of <code>value</code>  and <code>unit</code>. Is that ok with you, @jaimergp and @andreavolkamer?</p>

Reply:
I agree on using the precaluclated values, but from a teaching point of view, I think it is helpful for the reader to understand how these measurements can be converted, so could we leave a description on this in?

Reply:
It turns out, the `standard_units` allow multiple units:

> Why are there so many different types of Standard Units in the database?
> The published units are taken directly from the literature and we then attempt to standardise these to report as a standard type. This is an ongoing curation task, which was started with the most common units first.

https://chembl.gitbook.io/chembl-interface-documentation/frequently-asked-questions/chembl-data-questions#why-are-there-so-many-different-types-of-standard-units-in-the-database

This is why I find multiple standard units `['nM' 'ug.mL-1' '/uM']`, while I did expect only to find `nM`.

Thus, we will need the conversion step anyways.

Please do correct me, if I am overseeing or misunderstanding something here.

Reply:
I dropped all non-nM entries.